### DISCUSSION TITLE
Rules in creation Notice in enotices2.preview.ted.europa.eu

### DISCUSSION THREAD

**Discussion by user6330** [OP-TED/eForms-SDK#387] [Q&A] [answered]
Hello everyone,

using **https://enotices2.preview.ted.europa.eu/create-notice** i've notice 2 main difference that I can't understand.
I used noticesubtype = 1 for the example

1)  How they cluster **"GR-Buyer"**, **"GR-Procedure"** under **"Notice Sections"** ? Why they divided **"Notice Sections"** and **"Organizations"**?
Are there some other info where I can get the clusters?
![image](https://user-images.githubusercontent.com/127392893/227961436-bde3fe56-de66-4f0f-b852-e96317f7fc81.png)
![image](https://user-images.githubusercontent.com/127392893/227961524-82b89daf-52df-4ba1-9bfd-f6d9e0857c7c.png)
![image](https://user-images.githubusercontent.com/127392893/227961587-ec46db83-70c7-49e0-92d6-be0e8a3cd776.png)

2) Why the content **"GR-Change"** is missing? I can't find information or logic about the content if it should be hidden or not.
`{
    "id": "GR-Change",
    "contentType": "group",
    "displayType": "SECTION",
    "description": "Change",
    "_label": "group|name|GR-Change",
    "content": [{
            "id": "BT-140-notice",
            "contentType": "field",
            "displayType": "COMBOBOX",
            "description": "Change Reason Code",
            "_label": "field|name|BT-140-notice",
            "readOnly": true
        }, {
            "id": "BT-762-notice",
            "contentType": "field",
            "displayType": "TEXTAREA",
            "description": "Change Reason Description",
            "_label": "field|name|BT-762-notice"
        }, {
            "id": "BT-758-notice",
            "contentType": "field",
            "displayType": "TEXTBOX",
            "description": "Change Notice Version Identifier",
            "_label": "field|name|BT-758-notice",
            "readOnly": true
        }, {
            "id": "GR-ChangeSections",
            "contentType": "group",
            "nodeId": "ND-Change",
            "displayType": "GROUP",
            "description": "Information associated to a single Change (Section, Description & Procurement Documents)",
            "_label": "group|name|ND-Change",
            "_repeatable": true,
            "content": [{
                    "id": "BT-13716-notice",
                    "contentType": "field",
                    "displayType": "TEXTBOX",
                    "description": "Change Previous Section Identifier",
                    "_label": "field|name|BT-13716-notice",
                    "_repeatable": true
                }, {
                    "id": "BT-141(a)-notice",
                    "contentType": "field",
                    "displayType": "TEXTAREA",
                    "description": "Change Description",
                    "_label": "field|name|BT-141(a)-notice"
                }, {
                    "id": "BT-718-notice",
                    "contentType": "field",
                    "displayType": "RADIO",
                    "description": "Change Procurement Documents",
                    "_label": "field|name|BT-718-notice"
                }, {
                    "id": "BT-719-notice",
                    "contentType": "field",
                    "displayType": "TEXTBOX",
                    "description": "Change Procurement Documents Date",
                    "_label": "field|name|BT-719-notice"
                }
            ]
        }
    ]
}
`

Regards,
Ferdinand Junior Maddela

**Comment by elerik** [comment, marked as answer]
Well i split it in similar way at my web application. GR-Change is used (iam not expert at EU forms so this is only my guesses and understandings from documentation so may be someobdy can correct me) when already existing submited notice is and u want to change it. Than you have to fill fields from GR-Change Section because they are requested as references and so on. But if you create notice then you dont need it in your form and thats the reason why you cant see it at preview. 
https://docs.ted.europa.eu/eforms/latest/schema/change-notice.html

Organisation, its just splited in sample preview app. Its up to you if you render it in that way or in another. I took this as inspiration and use this in my app too. But you can for example ignore sections at all during render and create all Groups in seperated steps. Or you can draw each section in separated accordion. Whatever, you know :) its just, freestyle.

**Reply by user6330** [reply]
Hi Elerik,

thanks for your reply.
About Organisation ok, so I will freestyle it.
About GR-CHANGE, you said "understandings from documentation", which part of the documentation?

**Comment by elerik** [comment]
So i made for you this compilation of information. 

**About Notice Change** - https://docs.ted.europa.eu/eforms/latest/schema/change-notice.html
"As indicated previously, the purpose of change notices is multifold:

**Correct a value reported in the previous Change notice, or if this is the first Change notice, the original notice**,

Inform about changes to information not inside the notice (e.g. changed procurement documents),

While changes may apply to notices of any form type, the Change notice Form Type and Notice Type values must respectively correspond to "change" and "corr". A change notice may only concern a single notice and contains all the information from that initial notice with applied changes in addition to the information on those changes."

**About when Change is acceptable** - https://docs.ted.europa.eu/home/eforms/FAQ/index.html#_forms_and_procedures
When can I stop publication of a notice via the API?

Only when the notice is in status “SUBMITTED”. Once the status of the notice has changed to "PUBLISHING" or "PUBLISHED", it is no longer allowed to perform this action. When a submitted notice has entered the daily export to TED and OP has initiated the process of publication of the next OJ S (which happens around 16:00 CET on workdays), its status will change to “PUBLISHING” and subsequently to “PUBLISHED” (once published in TED). **In this case it will only be possible to submit a change notice for publication in the OJ S cancelling the initial notice, i.e. by creating a change notice with ReasonCode “cancel” from change-corrig-justification.gc. Both the original notice and its change notice will be published in the OJ S.**
